cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/ConsentWebView.java

```diff
@@ -93,16 +94,21 @@ private void setup() {
         this.setBackgroundColor(Color.TRANSPARENT);
         setWebViewClient(new WebViewClient() {
             @Override
-            public void onPageFinished(WebView view, String url) {
-                super.onPageFinished(view, url);
-                //view.loadUrl("javascript:" + "addEventListener('message', SDK.onEvent('oie'))");
+            public void onPageStarted(WebView view, String url, Bitmap favicon) {
+                super.onPageStarted(view, url, favicon);
                 try {
                     view.loadUrl("javascript:" + getFileContent(getResources().openRawResource(R.raw.js_receiver)));
                 } catch (IOException e) {
                     ConsentWebView.this.onError(new ConsentLibException(e, "Unable to load jsReceiver into ConasentLibWebview."));
                 }
             }
 
+            @Override
```

Comment:
do we have to keep this overriden method here?

Reply:
Thank you @leodoin

Reply:
> do we have to keep this overriden method here?

I will test by removing this overridden method , I believe we dont require this method.
I will let you know once testing is done.

Reply:
@leodoin Updated javascript injection logic and removed unused